pkg/cmd/server/start/start_master.go

```diff
@@ -558,6 +558,7 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunEndpointController()
 		kc.RunNamespaceController()
 		kc.RunPersistentVolumeClaimBinder()
+		kc.RunPersistentVolumeProvisioner(pvKClient)
```

Comment:
Not sure if you're referring to the pods that get launched or not but the controller itself needs permissions.  From what it looks like this controller needs permissions for CRUD on PVs, PV status, and PVCs which is different from the recycler which also includes permissions for pods and events.  Is that an incorrect summary?